static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-sign-in-cta.js

```diff
@@ -13,8 +13,8 @@ export const acquisitionsBannerSignInCtaTemplate = (
             <div class="engagement-banner__roundel">
                 ${marque36icon.markup}
             </div>
-            <button class="button engagement-banner__close-button js-site-message-close js-engagement-banner-close-button" data-link-name="hide release message">
-                <span class="u-h">Close</span>
+            <button tabindex="5" class="button engagement-banner__close-button js-site-message-close js-engagement-banner-close-button" data-link-name="hide release message">
```

Comment:
Why is tabindex 5 here and 4 elsewhere?

Reply:
this version of the banner has 3 CTAs (so using 3, 4, and 5)